src/components/Cards/Cards.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from '../Card/Card';
+import { CardsContainer } from '../../assets/styles/components/cards';
+
+const Cards = ({ data }) => {
+  const videos = data.filter((video) => video.id.kind.includes('#video'));
+  return (
+    <CardsContainer>
+      {data.length
+        ? videos.map((item) => <Card key={item.id.videoId} item={item} />)
```

Comment:
it would be better to check for `videos.length` since you're mapping on the filtered `videos`